io.greenfire.Greenery.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "io.greenfire.Greenery",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "23.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "greenery",
+    "separate-locales": false,
+    "tags": [
+        "proprietary"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
```

Comment:
why does it need sound access?

Reply:
Audio access is needed to trigger an audio sound (cash register)  when a wallet has a balance come through.